frontend/www/js/omegaup/components/contest/Edit.vue

```diff
@@ -21,6 +21,7 @@
         <a
           v-if="!virtual"
           href="#new_form"
+          data-contest-new_form
```

Comment:
Nit:
```suggestion
          data-contest-new-form
```
just for consistency with the rest of data-attributes.